contextily/tile.py

```diff
@@ -74,6 +75,7 @@ def bounds2raster(
     ll=False,
     wait=0,
     max_retries=2,
+    num_parallel_tile_downloads=16,
```

Comment:
```suggestion
    n_connections=16,
```
Something shorter like this would be preferable.